app/controllers/autotune/projects_controller.rb

```diff
@@ -183,6 +185,35 @@ def update
       end
     end
 
+    def preview_build_data
```

Comment:
It was the only way that I could figure out to get this to work when creating a new project since the project itself doesn't actually exist yet. It's mapped to:
```
match 'blueprints/:id/new_project/preview_build_data',
      :to => 'blueprints#preview_build_data',
      :constraints => { :id => Autotune::SLUG_OR_ID_REGEX },
      :via => [:get, :post]
```

Would love to know if there is a better way to do this so that I wouldn't have to duplicate.